appng-xmlapi/src/main/xsd/appng-platform.xsd

```diff
@@ -771,6 +771,7 @@ Therefore, several {@link Template}s are used, which contain the nessecarry XSL
 			</xs:appinfo>
 		</xs:annotation>
 		<xs:sequence>
+			<xs:element name="messages" type="messages" minOccurs="0" />
```

Comment:
Please place messages after config, same as for <action>

Reply:
I've placed the messages after config